bin/mark_changepoints_in_json

```diff
@@ -173,24 +173,42 @@ class Segments(object):
             if math.fabs(self.segments[0].mean - self.segments[1].mean) < self.delta:
                 self._merge_segment(0)
 
+    def _is_segment_a_slowdown(self, index, pc_bound=0.1):
+        """Test whether mean of segment index is slower than the last segment
+        mean by >= pc_bound percent.
+        """
+        if self.segments[index].mean <= self.segments[-1].mean:
```

Comment:
This doesn't look right -- it means that if `segments[index].mean` is 0.100001 and `segments[-1].mean` is 0.100002, then this will return `False` immediately without doing the `pc_bound`% check?